src/app/missions/detail/detail.component.css

```diff
@@ -18,7 +18,7 @@
     margin-left: 0;
 }
 
-.mission-rewards ::ng-deep lux-slot {
```

Comment:
> this was not possible before because the item component encapsulated the slot component

So this work's because it's an immediate child now? Does this also work for child elements of `lux-slot` or do they still need `::ng-deep`?

Reply:
Yes. Child elements would still  need `::ng-deep`, but this is intended, as that would be presentation details that would rightly be encapsulated, in contrast to `<lux-gui-item>` , which was just fetching data and populating the slot component without adding to the presentation.